examples/Demo/FluentUI.Demo.Client/Documentation/Components/Image/Examples/FluentImageBorder.razor

```diff
@@ -0,0 +1 @@
+﻿<FluentImage Bordered="true" Source="https://picsum.photos/200"/>
```

Comment:
You can't see any difference with “normal” images. Can you use this to render “empty” and display the border?

```html
<FluentImage Bordered="true" Source="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==" Width="100px" Height="100px" />
```